docs/content/doc/features/comparison.en-us.md

```diff
@@ -48,7 +48,8 @@ _Symbols used in table:_
 | Integrated Git-powered wiki         | ✓                                                  | ✓    | ✓         | ✓         | ✓         | ✓ (cloud only) | ✘            |
 | Deploy Tokens                       | ✓                                                  | ✓    | ✓         | ✓         | ✓         | ✓              | ✓            |
 | Repository Tokens with write rights | ✓                                                  | ✘    | ✓         | ✓         | ✓         | ✓              | ✓            |
-| Built-in Container Registry         | [✘](https://github.com/go-gitea/gitea/issues/2316) | ✘    | ✘         | ✓         | ✓         | ✘              | ✘            |
+| Built-in Container Registry         | [✘](https://github.com/go-gitea/gitea/issues/2316) | ✘    | ✓         | ✓         | ✓         | ✘              | ✘            |
+| Built-in Package Registry           | ✓                                                  | ✘    | ✓         | ✓         | ✓         | ✘              | ✘            |
```

Comment:
Could we have more detail abouth which packages Gitea supports? And Container should be one of them?

Reply:
At the moment there is no container registry but yes, should be one entry.
Which package is supported is documented in [doc/packages/overview.en-us.md](https://github.com/go-gitea/gitea/pull/16510/files#diff-77c07ab67c30445268eef68a3876876504fc18da985ff3b7fb6a065f9e814c04)